app/controllers/applicant_results_controller.rb

```diff
@@ -0,0 +1,10 @@
+class ApplicantResultsController < ApplicationController
+  before_action :authenticate_administrator!
+  before_action :set_applicant
+
+  private
+
+  def set_applicant
+    @applicant = Applicant.find(params[:applicant_id])
+  end
+end
```

Comment:
Can this be handled by `ApplicantsController`? If not I wonder if we should treat this like the other search controllers?

Reply:
This isn't a search controller, it's a tabbed controller (like `campaign_budgets_controller.rb`)